build/azure-pipelines/darwin/sql-product-build-darwin.yml

```diff
@@ -122,7 +122,7 @@ steps:
       INTEGRATION_TEST_ELECTRON_PATH="$APP_ROOT/$APP_NAME/Contents/MacOS/Electron" \
       VSCODE_REMOTE_SERVER_PATH="$(agent.builddirectory)/azuredatastudio-reh-darwin" \
       ./scripts/test-integration.sh --build --tfs "Integration Tests"
-    displayName: Run integration tests (Electron)
+    displayName: Run core integration tests # {{SQL CARBON EDIT}} Rename to core for clarity
```

Comment:
Remove sql carbon edit for any `sql-*` yml files - those are ones that we own completely so no need to mark them as changed.

Reply:
Done.